about.css

```diff
@@ -0,0 +1,444 @@
+  flex-direction: column;
+  position: absolute;
+  top: 50px;
+  animation: 1.5s slidein;
+  z-index: 4;
+  left: -30px;
+  width: 100vw;
+  height: 100vh;
+}
+
+.menu-i {
+  padding: 20px;
+  border-bottom: 1px solid #fff;
+}
+
+.hamberger-menu > .menu-i > a {
+  text-decoration: none;
+  color: #fff;
+}
+
+@keyframes slidein {
+  from {
+    margin-left: 100%;
+    width: 300%;
+  }
+
+  to {
+    margin-left: 0%;
+    width: 100%;
+  }
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  padding: 60px 10px 40px 10px;
+  position: relative;
+  z-index: 2;
+}
+
+.header-overlay {
+  background-image: url('./image/bg-1.jpg');
+  background-color: rgba(255, 255, 255, 0.8);
+  background-size: cover;
+  background-position: center;
+  filter: blur(2px);
+  -webkit-filter: blur(2px);
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: -1;
+}
+
+.sub-header {
+  color: var(--primary-color-2);
+  font-size: 28px;
+}
+
+h1 {
+  color: var(--primary-color-2);
+  font-size: 40px;
+  font-weight: 900;
+  margin: 10px auto;
+  text-transform: uppercase;
+  z-index: 3;
+}
+
+.description-h {
+  color: var(--primary-color-1);
+  border: 2px solid #fff;
+  background-color: rgb(248, 247, 247);
+  box-shadow: rgba(0, 0, 0, 0.05) 0 6px 24px 0, rgba(0, 0, 0, 0.08) 0 0 0 1px;
+  font-size: 14px;
+  line-height: 26px;
+  margin: 15px;
+  padding: 15px;
+  z-index: 5;
+}
+
+.time-schedule span {
+  font-size: 30px;
+  font-weight: 900;
+  color: var(--primary-color-1);
+}
+
+.pr-overlay {
+  background: linear-gradient(45deg, #040404 10%, transparent 11%, transparent 89%, #040404 90%), linear-gradient(135deg, #040404 10%, transparent 11%, transparent 89%, #040404 90%), radial-gradient(circle, transparent 25%, #232323  26%), linear-gradient(45deg, transparent 46%, #040404 47%, #040404 52%, transparent 53%), linear-gradient(135deg, transparent 46%, #040404 47%, #040404 52%, transparent 53%);
+  background-size: 3em 3em;
+  background-color: #232323;
+  opacity: 1;
+  filter: blur(2px);
+  -webkit-filter: blur(2px);
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  z-index: -1;
+}
+
+/* About */
+.description {
+  color: var(--primary-color-1);
+  border: 1px solid #888;
+  background-color: #fff;
+  line-height: 26px;
+  margin: 15px;
+  padding: 15px 5px;
+  z-index: 5;
+}
+
+.contact-info {
+  line-height: 26px;
+  margin: 20px 30px 0 30px;
+}
+
+.email {
+  font-weight: bold;
+  text-decoration: underline;
+}
+
+.emphasic,
+.courses-header {
+  color: var(--primary-color-1);
+  display: flex;
+  flex-direction: column;
+  line-height: 26px;
+  margin: 40px 15px;
+  text-align: center;
+}
+
+.emphasic h3 {
+  margin: 0 30px 10px 30px;
+}
+
+.emphasic hr,
+.courses-header hr {
+  border: none;
+  border-top: 1px solid var(--primary-color-2);
+  margin: 10px auto;
+  padding: 0;
+  width: 20px;
+}
+
+footer > div > p {
+  font-size: 12px;
+}
+
+.emphasic p {
+  margin: 0 30px;
+}
+
```

Comment:
- [ ] Great work adjusting the size of the logo as directed by the previous reviewer. However, the block of text was not worked on. The spacing on either side of the block of text needs to be identical to that of the template chosen. To achieve this, I would advise that you increase the `margin-right` and `margin-left` for desktop screen sizes. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/211144862-62513b32-b14e-4ee1-9bf8-52cfc81072a4.png)

### Template Design

![image](https://user-images.githubusercontent.com/77106962/211144879-c54e1af8-8375-42a7-af80-3af5d24e9109.png)

#### Hint: You can use a `margin` of `0 250px` for desktop screen sizes. 💻